modules/socialApi.js

```diff
@@ -0,0 +1,74 @@
+// socialApi.js
+
+const apiAddress = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps';
+const likesUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/4jsLQspdbzdaBFGyapdL/likes';
+const unqKey = '4jsLQspdbzdaBFGyapdL';
+
+const pushCommentToApi = async (pushThisToApi) => {
+  const outcome = await fetch(`${apiAddress}/${unqKey}/comments`, {
```

Comment:
- [ ] There should be a popup when I click comments where users can comment. Currently I cant do that

Reply:
hi @GzimAsani, thank you for your feedback.
There is not enough time to do this since I'm the only one that works on the project.
Thank you for your understanding.
I will complete that during my free time. thanks

Reply:
Hi @Ol-create, As requested by the previous reviewer, kindly implement the comment pop-up because it is required in the project. I know you said you worked solo which is understandable but then we have to follow the code review guideline and before I requested these changes, I confirmed with a senior code reviewer. You can write to student success about this or request a second opinion review.